package.json

```diff
@@ -31,15 +31,19 @@
   "homepage": "https://mobx-react.js.org",
   "peerDependencies": {
     "mobx": "^4.0.0 || ^5.0.0",
-    "react": "^16.8.0"
+    "react": "^16.8.0",
+    "babel-plugin-macros": "^2.6.1"
```

Comment:
I think it's better to document this instead of annoying everyone with peer dependency warning.